### DISCUSSION TITLE
Logfile per Connection?

### DISCUSSION THREAD

**Discussion by DocMAX** [strongswan/strongswan#2299] [General]
Is it possible to create logfiles per connection? I find it difficult to read the syslog.log if multiple connections are going on. They aren't separated in any way (no tags etc,)

**Comment by tobiasbrunner** [comment]
No, that's not possible. However, the log messages can be prefixed with the name of the connection (if it's available, otherwise it's just the ID of the IKE_SA or no identifier at all). See [the docs](https://docs.strongswan.org/docs/5.9/config/logging.html) for details.
